sdk/test-utils/recorder/package.json

```
@@ -56,26 +59,26 @@
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=8.0.0"
```

Is there a reason to go back and support node 8?
```suggestion
    "node": ">=12.0.0"
```